### PR TITLE
Added inprogress, updated styles, cleaned css

### PR DESCRIPTION
**Pull request by bmiddha**
- added effects for ongoing exams
- exam cards get auto archived once exam has ended
- new color scheme and styles
- css cleanup
- fixed bug when prepending "0" in single digit seconds

### CODE REVIEW

File: views/index.mst [hunk 1]
```diff
@@ -57,94 +45,107 @@ html,body {
...
+                        times[i].past = (times[i].end.dayOfYear() < now.dayOfYear());
+                        fc.innerHTML = "00:00:00";
+                    }
                     return;
                 }
+                if (_hrs < 2) {
+                    fc.parentElement.classList.add("countdownWarning")
+                }
+                
+                let hrs = diff.hours();
```
Comment (wilt00) [inline views/index.mst]: This is incorrect. .hours() returns the number of hours modulo 24 (ie subtracting off full days). Needs to be .asHours()

File: views/index.mst [hunk 2]
```diff
@@ -57,94 +45,107 @@ html,body {
...
             }
-        });
-
-        function updateCountdowns() {
-            const now = moment();
-            Array.from(document.getElementsByClassName("finalCountdown")).map((fc, i) => {
+            });
+    function updateCountdowns() {
+        const now = moment();
+        Array.from(document.getElementsByClassName("finalCountdown")).map((fc, i) => {
```
Comment (wilt00) [inline views/index.mst]: Stop putting garbage in my diffs!

File: views/index.mst (hunk 1 above)
Comment (bmiddha) [reply views/index.mst]: .asHours() breaks it. it works with .hours()
image
with asHours()
image
with hours()

File: views/index.mst (hunk 2 above)
Comment (bmiddha) [reply views/index.mst]: fixed

**Comment by bmiddha** [comment]
added meta theme color

File: views/index.mst (hunk 1 above)
Comment (wilt00) [reply views/index.mst]: I can't paste a picture on mobile, but look at Thursday and Friday's classes. The number of hours is wrong.

Comment (wilt00) [reply views/index.mst]: .asHours returns a float, you need to round it down

File: README.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# UIC Finals Dashboard
+UIC ACM finals countdown wall, rewritten for simplicity
+## Data Extraction
+This is done by `scrapper.ts`
```
Comment (wilt00) [inline README.md]: 'scraper'

File: views/index.mst (hunk 1 above)
Comment (bmiddha) [reply views/index.mst]: fixed

File: views/index.mst [hunk 4]
```diff
@@ -115,41 +104,54 @@ html,body {
...
+                        times[i].past = (times[i].end.dayOfYear() < now.dayOfYear());
+                        fc.innerHTML = "00:00:00";
+                    }
                     return;
                 }
+                if (_hrs < 2) {
+                    fc.parentElement.classList.add("countdownWarning")
+                }
+                
+                let hrs = Math.floor(diff.asHours());
```
Comment (wilt00) [inline views/index.mst]: This is now the same as _hrs above

**Pull request merged**